config/config.json

```diff
@@ -9,10 +9,11 @@
     },
     "policies": {
         "version": "1.0.0",
-        "artifactVerificationPolicies": {
```

Comment:
given this is a significant change to the policy structure, should bump the version? What is our backwards compat requirement,  or is this something we only need to consider after stable release?

Reply:
good point. I'm not sure. I just left it as is. I can bump it up, but honestly I don't think what we had before was really a previous version since it basically wasn't used.